FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +23,45 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure(_):
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
+			case .success((let data, let httpResponse)):
+				if httpResponse.statusCode != Constants.successSatusCode {
+					completion(.failure(RemoteFeedLoader.Error.invalidData))
+				} else {
+					if let images = try? JSONDecoder().decode(FeedLoaderResponse.self, from: data) {
+						completion(.success(images.items.compactMap { $0.feedImage }))
+					} else {
+						completion(.failure(RemoteFeedLoader.Error.invalidData))
+					}
+				}
+			}
+		}
+	}
+}
+
+struct FeedLoaderResponse: Decodable {
+	let items: [Image]
+}
+
+struct Image: Decodable {
```

Comment:
`Image` is an implementation detail of `RemoteFeedLoader`, so it should be marked as `private`:
```suggestion
private struct Image: Decodable {
```